### PR TITLE
Added passport provider for ru_RU language

### PR DESCRIPTION
**Pull request by Abdujabbar**
### What does this change

Added new passport provider for ru_RU language

### What was wrong

It looks like based on an issue https://github.com/joke2k/faker/issues/2061 for passport_owners we need to return first surname and then given_name

### How this fixes it

I created a new provider class in the path: faker/providers/passport/ru_RU/__init__.py and overloaded the function passport_owner

### CODE REVIEW

File: faker/providers/passport/ru_RU/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from typing import Tuple
+
+from .. import Provider as PassportProvider
+
+
+class Provider(PassportProvider):
+    def passport_owner(self, gender: str = "M") -> Tuple[str, str]:
```
Comment (fcurella) [inline faker/providers/passport/ru_RU/__init__.py:7]: you can use `faker.typing.SexLiteral` as type for the `gender` argument.

**Review by fcurella — CHANGES_REQUESTED** [review]

File: faker/providers/passport/ru_RU/__init__.py (hunk 1 above)
Comment (Abdujabbar) [reply faker/providers/passport/ru_RU/__init__.py:7]: Agree, let I will fix it! 

Thanks for feedback.

**Review by fcurella — CHANGES_REQUESTED** [review]
Can you add a test for the new provider?

**Comment by Abdujabbar** [comment]
> Can you add a test for the new provider?

@fcurella  I have added a simple use case for testing, let's try to check. 

But for now it looks like some pipelines failed, I guess it's not related into my changes, am I right?

**Review by fcurella — APPROVED** [review]
Thank you!

**Pull request merged**